docs/netbsd/found_bugs.md

```diff
@@ -6,4 +6,7 @@ mailing list and are listed on the [dashboard](https://syzkaller.appspot.com/#ne
 
 Newer bugs come first
 
+- connect(2): missing length check in some protocols causes a out of bounds read (Detected with KASan) - [fix](https://github.com/NetBSD/src/commit/a6926e46f91619f5a231a35b7886dd6c54a65ab3)
```

Comment:
Elide "Detected with"? You are likely to have dozens of them. Duplication will accumulate.

Reply:
Okay